.github/workflows/scala.yml

```diff
@@ -86,9 +86,13 @@ jobs:
 
       # Build
       - name: Bootstrap Enso project
-        run: sbt --no-colors bootstrap
+        run: |
```

Comment:
If this has helped you need to update the release workflow as well.

Reply:
This is a terrible hack, but it does the job. I was restarting this build multiple times yesterday, and have not encountered this particular error. 

Top errors by the number of occurrences:

8 - Out of memory (all-windows)
7 - Failed ProjectManagementApiSpec (4-linux, 3-mac)
2 - Cannot install GraalVM MBean due to null. java.lang.NoClassDefFoundError: Could not initialize class java.lang.ProcessEnvironment (all-mac)

Reply:
@radeusgd had an idea about the OOM during the native image build.